### DISCUSSION TITLE
is there a way to migrate all old version package to verdaccio？

### DISCUSSION THREAD

**Discussion by theinaku** [verdaccio/verdaccio#4699] [Q&A]
because we plan to close the old registry and use verdaccio only, that means will lost to access the old version of packge

**Comment by hellorayza** [comment]
I recently migrated verdaccio across platforms and versions, which was a problem I encountered and solved

https://github.com/verdaccio/verdaccio/issues/3190#issuecomment-2223248208

One issue I have at the moment is that the migrated packages cannot use pnpm unpublish and may need to be removed manually
